src/foam/dao/index/TreeIndex.java

```diff
@@ -71,14 +64,14 @@ public Object put(Object state, FObject value) {
   public Object remove(Object state, FObject value) {
     return ((TreeNode) state).removeKeyValue((TreeNode) state, prop_, prop_.f(value), value, tail_);
   }
-
+ 
```

Comment:
I don't think there's an extra space here. I just put a blank line between the methods as you told me to do before. That's right, isn't it?